Client/MirObjects/ItemObject.cs

```diff
@@ -105,6 +119,29 @@ public override void DrawName()
             NameLabel.Draw();
         }
 
+        public void DrawGradeEff()
+        {
+            if (Grade > ItemGrade.Common)
+            {
+                switch (Grade)
+                {
+                    case ItemGrade.Rare:
+                        _effect.BackColour = Color.DeepSkyBlue;
+                        _effect.ForeColour = Color.DeepSkyBlue;
+                        break;
+                    case ItemGrade.Legendary:
+                        _effect.BackColour = Color.DarkOrange;
+                        _effect.ForeColour = Color.DarkOrange;
+                        break;
+                    case ItemGrade.Mythical:
+                        _effect.BackColour = Color.Purple;
+                        _effect.ForeColour = Color.Purple;
+                        break;
+                }
+                Libraries.DropEffects.DrawBlend(0 + _effect.Index, new Point(DrawLocation.X, DrawLocation.Y), _effect.BackColour, true, 10F);
```

Comment:
I'm not sure why you're drawing it manually here? Since you've already created an animated object capable of doing everything, you should be able to just call

_effect .Draw();